js_modules/dagit/packages/core/src/ui/Colors.tsx

```diff
@@ -0,0 +1,40 @@
+export const ColorsWIP = {
+  Dark: 'rgba(23, 22, 21, 1)',
+  White: 'rgba(255, 255, 255, 1)',
+  Gray900: 'rgba(35, 31, 27, 1)',
+  Gray800: 'rgba(58, 54, 49, 1)',
+  Gray700: 'rgba(82, 78, 72, 1)',
+  Gray600: 'rgba(107, 103, 98, 1)',
+  Gray500: 'rgba(134, 131, 127, 1)',
+  Gray400: 'rgba(161, 157, 153, 1)',
+  Gray300: 'rgba(189, 186, 183, 1)',
+  Gray200: 'rgba(218, 216, 214, 1)',
+  Gray100: 'rgba(245, 244, 242, 1)',
+  Gray50: 'rgba(250, 249, 247, 1)',
+  Gray10: 'rgba(35, 31, 27, 0.03)',
+  Blue1: 'rgba(14, 12, 167, 1)',
```

Comment:
It's slightly confusing that blue1 (smallest number) is the darkest blue but Gray10 (smallest number) is the lightest gray?  I wonder if Gray1 -> Gray90 (being lightest gray) would be more consistent?

Reply:
Good point, in the specs they're referred to as e.g. `Red 700`, `Red 500`, etc. I can just go with that.